packages/table/src/headers/header.tsx

```diff
@@ -390,8 +374,33 @@ export class Header extends React.Component<IInternalHeaderProps, IHeaderState>
             // both selection and reordering behavior.
             && selectedRegions.length === 1;
     }
+}
 
-    private isSelectedRegionRelevant = (selectedRegion: IRegion) => {
-        return Regions.getRegionCardinality(selectedRegion) === this.props.fullRegionCardinality;
+/**
+ * Due to reasons we're still investigating, our tests fail if we put a common, custom
```

Comment:
this comment will probably be out of date as soon as monday. We're probably not going to continue investigating.

I suspect the reason those particular tests were failing is because the interactions to do re-ordering were adjusting state or props and causing some updates. The subsequent classname selector lookups wasn't able to find the right element. Or perhaps, a chain of new objects were created and the state was out of sync.

Either way, we should probably don't want to imply that we intend to change this in the future.